Source/Scene/BatchTable.js

```diff
@@ -169,6 +183,134 @@ define([
         return Number;
     }
 
+    function createOffsets(attributes, packFloats) {
+        var offsets = new Array(attributes.length);
+
+        var currentOffset = 0;
+        var attributesLength = attributes.length;
+        for (var i = 0; i < attributesLength; ++i) {
+            var attribute = attributes[i];
+            var componentDatatype = attribute.componentDatatype;
+
+            offsets[i] = currentOffset;
+
+            if (componentDatatype !== ComponentDatatype.UNSIGNED_BYTE && packFloats) {
```

Comment:
That would increase the memory even more. If the type is unsigned byte, we still only use one texel, but, if its a larger type, then we pack into unsigned bytes which is 4 texels.